File: app/src/main/java/com/OdiousPanda/rainbow/DataModel/Unsplash/Links_.java
package com.OdiousPanda.rainbow.DataModel.Unsplash;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Links_ {

    @SerializedName("self")
    @Expose
    public String self;
    @SerializedName("html")
    @Expose
    public String html;
    @SerializedName("photos")
    @Expose
    public String photos;
    @SerializedName("likes")
    @Expose
    public String likes;
    @SerializedName("portfolio")
    @Expose
    public String portfolio;
    @SerializedName("following")
    @Expose
    public String following;
    @SerializedName("followers")
    @Expose
    public String followers;

}

File: app/src/main/java/com/odiousPanda/rainbowKt/model/dataSource/unsplash/Links_.kt
package com.odiousPanda.rainbowKt.model.dataSource.unsplash

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Links_ {
    @SerializedName("self")
    @Expose
    var self: String = ""

    @SerializedName("html")
    @Expose
    var html: String = ""

    @SerializedName("photos")
    @Expose
    var photos: String = ""

    @SerializedName("likes")
    @Expose
    var likes: String = ""

    @SerializedName("portfolio")
    @Expose
    var portfolio: String = ""

    @SerializedName("following")
    @Expose
    var following: String = ""

    @SerializedName("followers")
    @Expose
    var followers: String = ""
}